spec/core_functions/content-exists/controls/controls.hrx

```diff
@@ -0,0 +1,28 @@
+<===> input.scss
+@mixin test-content-exists() {
+  @if content-exists() {
+    @content;
+  }
+  @else {
+    content-exists: false;
+  }
+}
+
+.should-be-true {
```

Comment:
Nit: I'd call these classes `.with-content` and `.without-content`. It's already clear what the expected result is, since it's right there in the output file, so it's more useful to document what's meaningfully different about how they're invoked.

Actually, when you split these into separate specs, you should probably put `with_content` and `without_content` into the test names and just make these selector names `a`.